### DISCUSSION TITLE
Push Notifications on Android

### DISCUSSION THREAD

**Discussion by zburlott4** [airsdk/Adobe-Runtime-Support#2757] [Q&A]
How does one go about adding push notifications for android. Is this natively supported in the SDKs? I haven't been able to find a guide anywhere how to do this for android.

**Comment by marchbold** [comment]
You can use our Push Notifications extension to implement them using FCM (or OneSignal) on Android: 

- docs: https://docs.airnativeextensions.com/docs/pushnotifications/
- license: https://airnativeextensions.com/extension/com.distriqt.PushNotifications
